packages/electron/src/main/BacktraceClient.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    BacktraceNodeClientSetup,
+    BacktraceSetupConfiguration,
+    BacktraceClient as NodeBacktraceClient,
+} from '@backtrace-labs/node';
+import { BacktraceClientBuilder } from './builder/BacktraceClientBuilder';
+import { BacktraceMainElectronModule } from './modules/BacktraceMainElectronModule';
+
+export class BacktraceClient extends NodeBacktraceClient {
+    constructor(clientSetup: BacktraceNodeClientSetup) {
+        super({
+            ...clientSetup,
```

Comment:
I think it's important to also use electron API to get application name and application version. In node we will try to understand the app structure and verify env, but it most likely won't work. 

https://www.electronjs.org/docs/latest/api/app#appgetversion

Reply:
Done!